### ISSUE TITLE
can I tag arrays opened within Maps?

### ISSUE THREAD

**Issue by mcr** [laurencelundblade/QCBOR#90]
if I do:
   QCBOREncode_OpenArrayInMapN(qec, keyint);

can I then call 
   QCBOREncode_AddTag(qec, MyTag);

to insert a tag *before* the array is started?

**Comment by mcr** [comment]
an experiment says no, so I have to open code this.

**Comment by laurencelundblade** [comment]
Yeah, it won't work. The tag will be applied to the integer map label. So you have to do this:
     QCBOREncode_AddInt64(qec, keyint);
     QCBOREncode_AddTag(qec, MyTag);
     QCBOREncode_OpenArray(qec);
